Aquality.Selenium.Core/src/Aquality.Selenium.Core/Elements/CachedElementStateProvider.cs

```diff
@@ -50,7 +50,15 @@ protected virtual bool TryInvokeFunction(Func<IWebElement, bool> func)
         public virtual void WaitForClickable(TimeSpan? timeout = null)
         {
             var errorMessage = $"Element {locator} has not become clickable after timeout.";
-            conditionalWait.WaitForTrue(() => IsClickable, timeout, message: errorMessage);
+            try
+            {
+                conditionalWait.WaitForTrue(() => IsClickable, timeout, message: errorMessage);
+            }
+            catch (TimeoutException e)
```

Comment:
discussed with @mialeska that it is necessary to keep the backward compatibility